Build.md

```diff
@@ -0,0 +1,35 @@
+## Run the demo
+This demo may not run directly on your device. So we have listed some of the problems you may encounter here.
```

Comment:
```suggestion
# How to build Taichi Unity examples

Because Taichi AOT is in active development, the Unity examples might suffer from unintentional breakage. This documentation helps you to recover from the incidents if we cannot immediately address your issue in the first place.

```